docs/spec/spec.md

```diff
@@ -2758,9 +2763,215 @@ public function main() returns error? {
 }
 ```
 
-## 13. Tools
+## 13. Federation
 
-### 13.1 GraphiQL client
+### 13.1 Federated Subgraph
+
+The Ballerina GraphQL module provides the capability to expose a `graphql:Service` as a [federation2 subgraph](https://www.apollographql.com/docs/federation/subgraph-spec). To convert a Ballerina GraphQL service into a federation2 subgraph, the `graphql.subgraph` sub module must be imported.
+
+```ballerina
+import ballerina/graphql.subgraph;
+``` 
+
+> **Note:** The current implementation of the subgraph only supports dynamic schema composition through introspection.
```

Comment:
We can generate the subgraph schema using the graphql tool and use the schema, right?

Reply:
Generating subgraph schema via graphql tools is not yet supported. Annotation (federated directive) values read at runtime.